src/sgame/sg_admin.cpp

```diff
@@ -5887,6 +5888,43 @@ static bool BotFillCmd( gentity_t *ent, const Cmd::Args& args )
 	return true;
 }
 
+static bool BotInfoCmd( gentity_t *ent, const Cmd::Args& args )
+{
+	int argc = args.Argc();
+	if ( argc < 3 )
+	{
+		BotUsage( ent );
+		return false;
+	}
+	int botIds[ MAX_CLIENTS ] = { 0 };
+	for ( int i = 2; i < argc && i - 2 < MAX_CLIENTS; i++ )
+	{
+		char err[ MAX_STRING_CHARS ];
+		botIds[ i - 2 ] = G_ClientNumberFromString( args[ i ].data(), err, sizeof( err ) );
+		if ( botIds[ i - 2 ] < 0 )
+		{
+			ADMP( va( "%s %s %s %s", QQ( "^3$1$:^* $2t$: $3$" ), "bot", Quote( err ), Quote( args[ i ].data() ) ) );
+			return false;
+		}
+	}
+	ADMBP_begin();
+	ADMP( QQ( N_( "Slot Name Team [s=skill ss=skillset b=behavior g=goal]" ) ) );
```

Comment:
Should this be ADMBP?

Reply:
I think this is the only strictly required change. Not changing the order in which description is shown would be nice, but it's much less important than having listbots readable.